.github/workflows/release-build.yml

```diff
@@ -1,36 +1,61 @@
 name: "Release Build"
 
 on:
+  workflow_dispatch:
   push:
-    tags:
-      - "v*"
-
+    branches:
+      - main
+      - dev
+      
```

Comment:
@BenjaminHalko 

It run on every commits, can you make it so that it won't run when you pushed to documentation or other miscellaneous files (e.g. documentation, license, ci)

Reply:
fixed in: #1664